test/e2e-olm/utils.go

```diff
@@ -366,3 +350,38 @@ func getCertificateContents(name, uuid string) []byte {
 	filename := fmt.Sprintf("%s/%s/%s", exampleSecretsPath, uuid, name)
 	return utils.GetFileContents(filename)
 }
+
+func getProjectRootPath(projectRootDir string) string {
```

Comment:
Is this function still used?

Reply:
It is still being used to find the root directory of the repo because go test does not support `-root` (The previous one is from the test package from operator-sdk).